src/resources/views/crud/columns/switch.blade.php

```diff
@@ -0,0 +1 @@
+@include('crud::columns.check')
```

Comment:
I think this column should be more similar to the field, showing a "pill" that resembles more the on/off switch.
**Let's leave this for the last and not work on this right now.**